endpoints.tf

```diff
@@ -68,3 +68,18 @@ data "template_file" "local_metastores_yaml" {
     mapped_databases = "${ lookup(var.local_metastores[count.index],"mapped-databases","") }"
   }
 }
+
+resource "aws_route53_zone" "remote_metastore" {
+  count = "${ var.enable_remote_metastore_dns == "" ? 0 : 1 }"
+  name = "${local.remote_metastore_zone_prefix}-${var.region}.lcl"
```

Comment:
Can we pass`.lcl` through variable?

Reply:
this entires are not to access waggledance, for accessing remote metastores bypassing waggledance.